packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/NftForm.hooks.ts

```diff
@@ -9,21 +14,56 @@ export const useNftForm = () => {
   const [listingType, setListingType] = useState<Listing>('Auction')
   const [currentStep, setCurrentStep] = useState(0)
 
+  const { convertDurationToBlocks } = useBlockTimeEstimation()
+
   const getTotalDaysAndHoursText = (start: Date, end: Date) => {
     const { days, hours } = intervalToDuration({
       start,
       end,
     })
-    const parsedDays = days ? `${days} ${days > 1 ? 'Days' : 'Day'}` : ''
-    const parsedHours = hours ? `${hours} ${hours > 1 ? 'Hours' : 'Hour'}` : ''
+
+    const parsedDays = days ? pluralizeNoun(days, 'Day') : ''
+    const parsedHours = hours !== undefined ? (hours >= 1 ? pluralizeNoun(hours, 'Hour') : 'Less than an hour') : ''
```

Comment:
This is a bit error prone, for example it can produce "Total: 8 Days Less than an hour" which isn't very correct. I think we don't need to do this manually, we can use for example `formatDistance` from `date-fns`